packages/cli/src/lib/bundler/server.ts

```diff
@@ -199,7 +203,26 @@ DEPRECATION WARNING: React Router Beta is deprecated and support for it will be
       root: paths.targetPath,
     });
   } else {
-    const compiler = webpack(config);
+    const publicPaths = await resolveOptionalBundlingPaths({
+      entry: 'src/index-public-experimental',
```

Comment:
Switched this around to a very explicitly separate and experimental entry point instead, so that we can ship it without risking conflicts with existing files.